modules/openxr/openxr_api.cpp

```diff
@@ -2319,6 +2322,10 @@ bool OpenXRAPI::pre_draw_viewport(RID p_render_target) {
 	return true;
 }
 
+XrCompositionLayerProjectionView *OpenXRAPI::get_projection_views() {
+	return render_state.projection_views;
+}
```

Comment:
Do we actually need this? I thought `set_projection_views_and_get_next_pointer()` was added so that we _wouldn't_ need to have this function (which would be problematic for multithreaded rendering).

This should be removed, if possible. If not, we'll need to go back to the drawing board :-)

Reply:
Ah good catch, I was still using this unnecessarily. I've removed it.